art/defences/detector/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+Module implementing detector-based defences against adversarial attacks.
+"""
+from art.defences.transformer import evasion
+from art.defences.transformer import poisoning
```

Comment:
When importing `from art.defences.detector import poisoning` I am actually importing the poisoning module from the sibling module `transformer` instead of `detector`. Is this intended? (same holds for `evasion`)

I assume it should be:
```
from art.defences.detector import evasion
from art.defences.detector import poisoning
```

Reply:
No, this was not intended and needs to be fixed.